server.js

```diff
@@ -19,8 +39,33 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec()
+  res.json(thoughts)
+})
+
+app.post('/thoughts', async (req, res) => {
+  const {message} = req.body
+  const thoughts = new Thought({message})
+
+  try {
+    const saveThought = await thoughts.save()
+    res.status(201).json(saveThought)
+  } catch (err) {
+    res.status(400).json({message: 'could not save task to db', error:err.errors})
+  }
+})
+
+app.post('/thoughts/:id/like', async (req, res) => {
+  const { id } = req.params
+  const findThought = await Thought.findOne({_id: id})
+
+  if (findThought) {
+    await Thought.updateOne({_id: id}, {$inc : {hearts: 1}})
+    res.status(201).json({message: `added like to ${id}`})
+  } else {
+    res.status(400).json({message: "can not find thought"})
```

Comment:
Consistency check here: Why use single quotes on line 65 and double qoutes on 67?